### PR TITLE
[web3torrent] Adds proper texts

### LINKED ISSUE #239: [web3torrent] Write proper copy for the UI
We should get rid of any placeholder text (i.e. Lorem Ipsum) and write useful text for the app's UI.

### PR DESCRIPTION
**Pull request by joelalejandro**
This PR removes any Lorem-ipsum-esque text on the app and replaces it with proper wording.

Resolves #239.

### CODE REVIEW

File: packages/web3torrent/src/pages/file/File.tsx [hunk 1]
```diff
@@ -61,8 +61,26 @@ const File: React.FC<RouteComponentProps> = () => {
...
+              <strong>Is it safe?</strong>
+              <br />
+              Web3Torrent operates with budgets; therefore, the app will <b>never</b> use any funds
+              outside whatever amount you allocate when starting the download. Also, Web3Torrent is
+              powered by{' '}
+              <a href="http://statechannels.org" target="_blank" rel="noopener noreferrer">
+                State Channels
+              </a>
+              , a technique that reducess fees for blockchain users and allows users to transact
+              with each other with instant finality. Through counterfactually instantiated
```
Comment (spersico) [inline packages/web3torrent/src/pages/file/File.tsx]: _instant finality_? What does _instant finality_ mean?

File: packages/web3torrent/src/pages/file/File.tsx [hunk 2]
```diff
@@ -61,8 +61,26 @@ const File: React.FC<RouteComponentProps> = () => {
...
+              contracts, this technology enables a private, efficient and secure environment for
+              safe transactions.
```
Comment (spersico) [inline packages/web3torrent/src/pages/file/File.tsx]: _safe_ is unnecessary here, IMHO, as you already say that you've got a secure environment.

File: packages/web3torrent/src/pages/file/File.tsx [hunk 3]
```diff
@@ -61,8 +61,26 @@ const File: React.FC<RouteComponentProps> = () => {
           </FormButton>
           <div className="subtitle">
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-              incididunt ut labore et dolore magna aliqua.
+              <strong>How do I pay for the download?</strong>
+              <br />
+              When you click "Start Download", you'll be asked to allocate an amount of ETH so
+              Web3Torrent can collect payments on your behalf and transfer those funds to the file
+              owner. Unlike other systems, the payment is not upfront; instead, you pay as you
```
Comment (spersico) [inline packages/web3torrent/src/pages/file/File.tsx]: I would replace __file owner__ with seeder, as:
- A Seeder of a file is not, necessarily the same as the file owner.
- Ownership is a pretty.... complicated word when we are talking about torrents. :-P

File: packages/web3torrent/src/pages/upload/Upload.tsx [hunk 4]
```diff
@@ -28,8 +28,16 @@ const Upload: React.FC<RouteComponentProps> = ({history}) => {
...
+          <strong>What kind of files can I share?</strong>
+          <br />
+          Any kind! Just make sure you're not sharing copyrighted material, or if you do, that
+          you're legally allowed to do so.
+        </p>
+        <p>
+          <strong>How do others download my file?</strong>
+          <br />
+          Use the "Share Link" button or copy this browser tab's URL, and send that address to
+          whoever you want to share the file with.
```
Comment (spersico) [inline packages/web3torrent/src/pages/upload/Upload.tsx]: _archer_ comment here

File: packages/web3torrent/src/pages/welcome/Welcome.tsx [hunk 5]
```diff
@@ -15,8 +15,12 @@ const Welcome: React.FC<RouteComponentProps> = ({history}) => {
...
       <div className="subtitle">
         <p>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
-          ut labore et dolore magna aliqua.
+          Web3Torrent offers a new experience for sharing files in a decentralized way via paid
+          streaming, bringing together{' '}
+          <a href="https://statechannels.org" target="_blank" rel="noopener noreferrer">
+            State Channels
+          </a>{' '}
+          and the BitTorrent protocol.
```
Comment (spersico) [inline packages/web3torrent/src/pages/welcome/Welcome.tsx]: mmm... should we add a shoutout to WebTorrent here? What do you think?

File: packages/web3torrent/src/pages/file/File.tsx (hunk 1 above)
Comment (joelalejandro) [reply packages/web3torrent/src/pages/file/File.tsx]: Replaced it with `faster-than-onchain operation times`.

File: packages/web3torrent/src/pages/welcome/Welcome.tsx (hunk 5 above)
Comment (joelalejandro) [reply packages/web3torrent/src/pages/welcome/Welcome.tsx]: Sure thing!

**Pull request merged**